flocker/node/functional/test_docker.py

```diff
@@ -367,6 +367,23 @@ def added(_):
         d.addCallback(added)
         return d
 
+    def test_null_environment(self):
+        """
+        A container that does not include any environment variables contains
+        an empty ``environment`` in the return ``Unit``.
+        """
+        client = self.make_client()
+        name = random_name()
+        self.addCleanup(client.remove, name)
+        d = client.add(name, u"busybox:latest")
```

Comment:
Using an image we control and can be sure actually has a null environment would make this test less likely to fail in the future when the busybox maintainers design to do something reasonable (like put some environment variables into their image).